include/openPMD/IO/AbstractIOHandler.hpp

```diff
@@ -101,8 +101,8 @@ class AbstractIOHandler
     virtual std::string backendName() const = 0;
 
     std::string const directory;
-    AccessType const accessTypeBackend;
-    AccessType const accessTypeFrontend;
```

Comment:
Is it on purpose that you did not rename those? `accessBackend` could me misleading, sounds like we're accessing the backend ;) Maybe `backendAccess`? Or just leave it